debian/changelog.in

```diff
@@ -2,6 +2,12 @@ wake (@VERSION@-1) unstable; urgency=medium
 
   * Auto-generated release package.
 
+ -- Ashley Coleman <ashley.coleman@sifive.com>  Fri, 28 Jul 2023 11:00:00 -0800
+
+wake (37.2.0-1) unstable; urgency=medium
+
+  * Auto-generated release package.
+
```

Comment:
As an aside, I'm wondering if we actually need to set this within a git commit, or if the release/CI script could just modify the file on the fly after reading the git tag value

Reply:
Maybe, its a  bit hard since its not the current tag but  instead the previous tag that gets inserted.

We could maybe regenerate the file in its entirety if we went back and tagged all the release commits on master. 

Automation here may also effect our ability to create one-off patch branches if we  ever needed that.

Idk, getting automation here would be nice and something I think is worth doing. The more we automate stuff like this the easier a fully automated release becomes

Reply:
> We could maybe regenerate the file in its entirety if we went back and tagged all the release commits on master.

I was thinking we could pretend each time that it's the first release